server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/TabletData.java

```diff
@@ -58,19 +58,18 @@ public TabletData(KeyExtent extent, VolumeManager fs, TabletMetadata meta) {
     this.flushID = meta.getFlushId().orElse(-1);
     this.directoryName = meta.getDirName();
     this.logEntries.addAll(meta.getLogs());
-    meta.getScans().forEach(tabletFile -> scanFiles
-        .add(new FileRef(fs, tabletFile.getMetadataEntry(), meta.getTableId())));
+    meta.getScans()
+        .forEach(tabletFile -> scanFiles.add(new FileRef(tabletFile.getMetadataEntry())));
```

Comment:
Would probably be useful to have a FileRef constructor that takes a TabletFile type.  This way its always converted easily and consistently.